### PR TITLE
fix: combobox scrolling top on web

### PR DESCRIPTION
**Pull request by matheusmendes-cb**
closes 

fixes 

Previous PR fixing bug where opening the address with keyboard open on mobile devices was causing the page to crash created another bug where the page is scrolled top on web also. This PR fixes this bug.

## How has it been tested?

- [x] Locally
- [x] Vercel Preview Branch
- [ ] Unit test
- [ ] Functional test

### CODE REVIEW

File: src/hooks/useGetElementDimensions.ts [hunk 1]
```diff
@@ -0,0 +1,14 @@
+import React from 'react'
+
+export function useGetElementDimensions(ref: React.RefObject<HTMLElement>) {
```
Comment (travisbloom-cb) [inline src/hooks/useGetElementDimensions.ts:3]: Why the change from `useResizeObserver`?

Comment (matheusmendes-cb) [reply src/hooks/useGetElementDimensions.ts:3]: It wasn't working, pretty much. When resizing the screen from mobile to web, the dialog on web was getting a width of `0px`. Now it behaves as expected, every time the input is resized, the dialog is resized as well

Comment (matheusmendes-cb) [reply src/hooks/useGetElementDimensions.ts:3]: Also, the safe useResizeObserver hook is quite more complex, [check it out](https://usehooks-ts.com/react-hook/use-resize-observer). I don't think we need something like that for now.

**Review by travisbloom-cb — APPROVED** [review]
